target/linux/ramips/image/mt7621.mk

```diff
@@ -413,6 +413,14 @@ define Device/dlink_dir-xx60-a1
 	check-size
 endef
 
+define Device/dlink_dir-1260-a1
+  $(Device/dlink_dir-xx60-a1)
+  DEVICE_PACKAGES += kmod-mt7603 kmod-mt7663u kmod-mt7663-firmware-ap
```

Comment:
```kmod-mt7615e``` also provides MT7663E (MT7613E) support. ```kmod-mt7663u``` is for USB wlan device, please drop.

https://github.com/openwrt/mt76/blob/93e3fce916c62d06892d41bf00d0f4c2926c9a0b/mt7615/Kconfig#L8